_maps/RandomRuins/SpaceRuins/oldstation.dmm

```diff
@@ -3204,9 +3204,7 @@
 /area/ruin/space/has_grav/ancientstation/engi)
 "ie" = (
 /obj/structure/table,
-/obj/item/stack/cable_coil{
-	amount = 30
```

Comment:
this isn't an active turf.

Reply:
Map merger doing stuff, you should find they are 30 by default hence the replacement.